packages/wdio-reporter/src/stats/runnable.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Main class for a runnable class (e.g. test, suite or a hook)
+ * mainly used to capture its running duration
+ */
+
+export interface Runnable {
+    uid?: string
```

Comment:
Should this be a string? Should it be optional?
See the getIdentifier method in this class.

Reply:
String